tests/test_benchmark_simple_scenarios.py

```diff
@@ -215,35 +214,36 @@ def test_benchmark_AE_grid_battery_peak_pricing(self, margs):
         # make the time the index
         busses_flow = busses_flow.set_index("Unnamed: 0")
         # read the columns with the values to be used
-        DSO_period = [
+        DSO_periods = [
             busses_flow["Electricity grid DSO_consumption_period_1"],
             busses_flow["Electricity grid DSO_consumption_period_2"],
             busses_flow["Electricity grid DSO_consumption_period_3"],
         ]
         demand = busses_flow["demand_01"]
         battery_charge = busses_flow["battery"]
+        # todo storage_discharge calculation should be replaced by timeseries as soon as that is stored to excel or if json is accessed
         battery_discharge = (
             abs(busses_flow["demand_01"])
             - busses_flow["Electricity grid DSO_consumption_period_1"]
+            - busses_flow["Electricity grid DSO_consumption_period_2"]
+            - busses_flow["Electricity grid DSO_consumption_period_3"]
         )  # todo: replace this by discharge column when implemented
-        # pick few instances
-        instances = [random.randint(1, len(DSO_period[1])) for x in range(0, 60)]
         # look for peak demand in period
         for j in range(0, 2):
```

Comment:
```suggestion
        for j in range(0, 3):
```
Otherwise, only period 1 and 2 are tested, not 3.